core/src/main/res/values/strings.xml

```diff
@@ -230,6 +230,7 @@
   <string name="severe_loss_error" tools:keep="@string/severe_loss_error">Severe error! Try Disable/Re-enable WiFi P2P</string>
   <string name="connection_failed" tools:keep="@string/connection_failed">Connection failed</string>
   <string name="permission_rationale_location">Location permission is required by Android to allow the app to detect peer devices</string>
+  <string name="permission_rationale_location_on_host_zim_file">Location permission is required by Android to allow the app to Host Zim files</string>
```

Comment:
End with period? or is the standard to not have periods in dialogs? My reference is the "Entering external link!" dialog. Also, should this be in `values-qq`? https://github.com/kiwix/kiwix-android/blob/develop/STRING_RESOURCES.md

Reply:
>End with period? or is the standard to not have periods in dialogs?

@Frans-Lukas  I don't know. @macgills can help with that.

>Also, should this be in values-qq as well?

Done.

Reply:
Typically no period so this is fine